.cargo/config

```diff
@@ -0,0 +1,21 @@
+# Cargo configuration
```

Comment:
I'm unsure if we really want this file - I feel like decising on the warning set is the job of the end binary, not of libraries.  Did this spot anything that was a genuine bug?

Reply:
I'm currently working on the assumption that this set of warning doesn't work up to users of the library. But I'm checking that.

Reply:
I have AMC confirmation that they don't get passed up